examples/h2o/random_forest.py

```diff
@@ -0,0 +1,32 @@
+import h2o
+from h2o.estimators.random_forest import H2ORandomForestEstimator
+
+import mlflow
+import mlflow.h2o
+
+h2o.init()
+
+wine = h2o.import_file(path="wine-quality.csv")
+r = wine['quality'].runif()
+train = wine[r < 0.7]
+test = wine[0.3 <= r]
+
+
+def trainRandomForest(ntrees):
```

Comment:
Nit: I'd change this to train_random_forest in order to follow the naming convention in the rest of the code. I guess this is based on the notebook somehow.

Reply:
done